Automattic/wp-calypso
# State: Use nock test helper consistently

This pull request seeks to update tests mocking network requests within the state directory to consistently use the `use-nock` test helper. The test helper provides conveniences for restoring default network behavior after the tests have run, which helps avoid unintended errors and reduces boilerplate. In the process of updating these tests, I had found several tests which were not properly cleaning up mocked requests after the test had finished.

__Testing instructions:__

There should be no impact on the Calypso application. Verify instead that tests continue to pass as expected:

```
npm test
```

Test live: https://calypso.live/?branch=update/state-nock-consistency
